schedule.py

```diff
@@ -0,0 +1,54 @@
+from typing import Iterable
+from flask_restful import abort
+
+from scheduledao import ScheduleDAO
+from event import Event
+
+class Schedule:
+    dao = ScheduleDAO()
+
+    def __init__(self, owner: str, events: Iterable[Event]=[]):
```

Comment:
Similar as the class structure above, take in a collection and have the class variables access that dictionary to apply the values .. the payload coming in will be from a `json` payload from the APP accepting an http Rest request, so it will just be passed into the constructor